BlueElectrum.js

```diff
@@ -34,17 +44,29 @@ async function connectMain() {
     });
 
     if (ver && ver[0]) {
-      console.log('connected to ', ver);
+      Sentry.addBreadcrumb({
+        category: 'BlueElectrum',
+        message: `connected to, ${ver}`,
+        level: Sentry.Severity.Info,
+      });
       mainConnected = true;
       wasConnectedAtLeastOnce = true;
     }
   } catch (e) {
     mainConnected = false;
-    console.log('bad connection:', JSON.stringify(usingPeer), e);
+    Sentry.addBreadcrumb({
+      category: 'BlueElectrum',
+      message: `bad connection: ${JSON.stringify(usingPeer)}, Error: ${e.message}`,
+      level: Sentry.Severity.Info,
```

Comment:
I understand that previously it was also a info level (`console.log`), however, since we're switching to Senetry it's a good opportunity to do it better and actually use the proper levels. Thus, I'd propose to change the log level to `Sentry.Severity.Error` or `Sentry.Severity.Warn` here and in other places like this

Reply:
yes, this would greatly improve sentry and information we aggregate with it.

Reply:
done